internal/database/sqlcommon/namespace_sql.go

```diff
@@ -90,6 +91,7 @@ func (s *SQLCommon) UpsertNamespace(ctx context.Context, namespace *core.Namespa
 				Set("name", namespace.Name).
 				Set("description", namespace.Description).
 				Set("created", namespace.Created).
+				Set("contract_index", namespace.ContractIndex).
```

Comment:
... sure I'll work this out as I go through, but reading this I assume this means "the last contract_index I detected an event from"